fetch-capability-statements.sh

```diff
@@ -12,7 +12,7 @@ function fetch_capability_statement {
 		status_code=$(curl -k -H 'Content-Type: application/fhir+json' -o /dev/null -sw '%{http_code}' ${url});
 		if [ ${status_code} -eq 200 ]
 		then
-			curl -k -H "Content-Type: application/fhir+json" -o ${IG_PATH}/fsh-content/input/resources/CapabilityStatement-${service}.json ${url}
+			curl -k -H "Content-Type: application/fhir+json" -o ${IG_PATH}/input/resources/CapabilityStatement-${service}.json ${url}
```

Comment:
Why has the url been changed?

Reply:
After the IG publishing pipeline was changed by trifork to conform to the community standard, this url was changed as par of the process.